ludwig/schema/features/augmentation/utils.py

```diff
@@ -0,0 +1,145 @@
+from dataclasses import field
+
+from marshmallow import fields, ValidationError
+
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.constants import TYPE
+from ludwig.schema import utils as schema_utils
+from ludwig.utils.registry import Registry
+
+_augmentation_config_registry = Registry()
+
+
+@DeveloperAPI
+def get_augmentation_config_registry() -> Registry:
+    return _augmentation_config_registry
+
+
+def register_augmentation_config(name: str):
```

Comment:
One further refinement of this registry: since `BaseAugmentationConfig` will eventually be the base of non-image augmentations as well, we want to have a one-to-many mapping of augmentation types to related feature types (rather than having one registry where everything is dumped). See `register_encoder_config` for an example of how to do this.

Reply:
done